book_source/developers_guide/Logging.Rmd

```diff
@@ -52,7 +52,7 @@ The second logging package is http://cran.r-project.org/web/packages/futile.logg
 
 To be able to use the loggers there needs to be some initialization done. Neither package allows to read it from a configuration file, so we might want to use the pecan.xml file to set it up. The setup will always be somewhat the same:
 
-```r
+```{r loggingexample1, echo=TRUE}
```

Comment:
Chunk ID comment below. Also, on the line after this, we use the `logging` library...but that's not actually what PEcAn uses, right? We should make sure that documentation is correct. Though I personally am strongly in favor of using somebody else's (probably better) logging solution over our current system.